patt/src/main/java/org/example/builder/Director.java

```diff
@@ -0,0 +1,16 @@
+package org.example.builder;
+
+public class Director {
```

Comment:
You have to format your code. You have code-styles problems that are not related to the logic of the code. Change it across all classes. It's important for real projects.

IntelliJ -> Code -> Reformat code.

To make it in all classes, 
<img width="421" alt="image" src="https://github.com/MarynaSkrypnyk/a-level/assets/72043323/c68386e0-2398-4588-9f6e-cdaf0390091a">
RMB (right mouse click on java folder)-> reformat code